### PR TITLE
Update UserGuide

### PR DESCRIPTION
**Pull request by kangzongxian**
Updated the UserGuide to reflect our GuestBook features and details

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -156,14 +164,14 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+GuestBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+GuestBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should we change the .json file to guestbook?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,23 +18,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists all guests.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com ci/19-05-2022 co/24-05-2022 ng/3`: Adds a guest named `John Doe` to the Guest Book.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should we make "Guest Book" consistent with the other descriptions. (i.e other descriptions uses "guest book") (see line 86, 104, 110, 141, etc)

**Review by jehousoh — CHANGES_REQUESTED** [review]
lgtm other than some minor issues.

File: docs/UserGuide.md (hunk 2 above)
Comment (kangzongxian) [reply docs/UserGuide.md]: Sounds good!

**Pull request merged**